src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -1757,6 +1889,68 @@ void AssimpImporterTest::lineMesh() {
     CORRADE_COMPARE(meshObject->instance(), 0);
 }
 
+void AssimpImporterTest::meshSkinningAttributes() {
+    auto&& data = ExportedFileData[testCaseInstanceId()];
+    setTestCaseDescription(data.name);
+
+    if(!supportsSkinning(data.suffix))
+        CORRADE_SKIP("Skin data for this file type is not supported with the current version of Assimp");
+
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("AssimpImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR,
+        "skin" + std::string{data.suffix})));
+    CORRADE_COMPARE(importer->meshCount(), 2);
+
+    /* Custom attributes should not be available until a mesh is imported.
+       These should return nothing (and not crash). */
+    CORRADE_COMPARE(importer->meshAttributeName(meshAttributeCustom(0)), "");
+    CORRADE_COMPARE(importer->meshAttributeName(meshAttributeCustom(564)), "");
+    CORRADE_COMPARE(importer->meshAttributeForName("thing"), MeshAttribute{});
```

Comment:
As per my comment above, `importer->meshAttributeForName("JOINTS")` *should* work here.